docs/learn.rst

```diff
@@ -211,3 +243,5 @@ If your teacher/coach scheduled an exam for you or your class, it will be availa
 .. figure:: img/exam-result.png
 	:alt: exams tab
 
+
+Here you will find learning topics and materials related to what you were doing the last time you used Kolibri, or those recommended by your teachers and coaches (not visible if you are browsing as a guest).
```

Comment:
This doesn't make sense in the context of exams?

![image](https://user-images.githubusercontent.com/374612/38116433-4bc17b9a-33b0-11e8-8516-f54932c60392.png)

Reply:
Ah! It seems to belong further up, moved!

Reply:
Totally true, thnx! 🤦‍♀️